### ISSUE TITLE
[HOLD for payment 2023-01-16] Manage members : Privacy policy link color changes hovering on it

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#14002]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app
2. Open Settings
3. Open Workspaces
4. Open any workspace
5. Open 'Manage Members'
6. Click on invite
7. Hover on 'Privacy Policy' link

## Expected Result:
Link color should remain same and not change on hover like other links in the app

## Actual Result:
Link color changes on hover to rgb(176,217,255) from rgb(90,176,255) 
(Inconsistency with other links on app, they don't have color change effect on hover)

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.48-2
**Reproducible in staging?:** y 
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/210680125-85b0b61e-5a3c-4619-a95f-ced9a8210be2.mp4

https://user-images.githubusercontent.com/43996225/210680139-792e5a5c-f637-430e-9a8e-b77b914f03c8.mov

**Expensify/Expensify Issue URL:**
**Issue reported by:** @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1672824339681399

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01c05e109d87455315</li>
        <li>Upwork Job ID: 1611127562018840576</li>
        <li>Last Price Increase: 2023-01-05</li>
    </ul>
</details>

**Comment by luacmartins** [comment]
[PR](https://github.com/Expensify/App/pull/14041) up

**Comment by luacmartins** [comment]
This was merged and deploy to prod 4 days ago.

**Comment by dhanashree-sawant** [comment]
Hi @luacmartins, the bug was kept on hold for payment with 16-01-2023 as payment date. Whenever you are available can you check it once?

**Comment by luacmartins** [comment]
@greg-schroeder can you please help with payment for this job?

**Comment by greg-schroeder** [comment]
Yes, doing now

**Comment by greg-schroeder** [comment]
https://www.upwork.com/jobs/~01c05e109d87455315

@dhanashree-sawant
@sobitneupane 

Please apply and I'll pay out ASAP

**Comment by Dhruval-bhansali** [comment]
Yes applied @greg-schroeder

**Comment by dhanashree-sawant** [comment]
Yes applied @greg-schroeder

**Comment by sobitneupane** [comment]
Thanks @greg-schroeder.

Applied for the job.

**Comment by greg-schroeder** [comment]
Okay, hired

**Comment by sobitneupane** [comment]
@greg-schroeder Accepted the offer.

**Comment by dhanashree-sawant** [comment]
@greg-schroeder Accepted the offer

**Comment by luacmartins** [comment]
Cool! Please close this issue once all payment is done!
